uberfire-extensions/uberfire-simple-docks/uberfire-simple-docks-client/src/main/java/org/uberfire/client/docks/view/DockResizeBar.java

```diff
@@ -33,9 +35,15 @@
 import com.google.gwt.user.client.ui.FlowPanel;
 import org.uberfire.client.resources.WebAppResource;
 import org.uberfire.client.workbench.docks.UberfireDockPosition;
+import org.uberfire.client.workbench.ouia.OuiaAttribute;
+import org.uberfire.client.workbench.ouia.OuiaComponent;
+import org.uberfire.client.workbench.ouia.OuiaComponentIdAttribute;
+import org.uberfire.client.workbench.ouia.OuiaComponentTypeAttribute;
 import org.uberfire.mvp.ParameterizedCommand;
 
-public class DockResizeBar extends FlowPanel {
+public class DockResizeBar extends FlowPanel implements OuiaComponent {
+
+    private static final String OUIA_COMPONENT_TYPE = "qe-resize-docks-bar";
```

Comment:
I wouldn't prefix with 'qe-' ... data-ouia is for use in testing, the prefix for the value is redundant.

Reply:
Agree, will remove it